_layouts/default.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
This layout doesn't seem to be referenced anywhere.

Reply:
I don't think I've added the.

The way this repo is setup, you're using / inheriting the default Jekyll minima theme. I need to include it in order to override it and have header `<link>` fields that will link to the all and last call feeds.

I'll either include this directly so it's clear or remove this for a future PR.